03/app.js

```diff
@@ -1,3 +1,13 @@
 export default function randomNumber(min, max) {
+  if (typeof min !== 'number' || typeof max !== 'number') {
+    throw new Error('Argumenty muszą być liczbami');
+  }
```

Comment:
Warto sprawdzić osobno i podać konkretniejszy komunikat tj. pierwsza/druga liczba jest niepoprana. Wtedy przy testowaniu też czytelniej to wygląda.